source/fundamentals/logging.txt

```diff
@@ -10,216 +10,21 @@ Logging
    :depth: 1
    :class: singlecols
 
-Overview
---------
+.. note::
```

Comment:
```suggestion
.. important::
```

Suggestion: I think this is more than [supplementary information](https://docs.mongodb.com/meta/style-guide/style/callouts/), so I think this should be `important`. I don't think readers should skip over this.